nsis/SharedBase.nsh

```diff
@@ -38,7 +38,8 @@ SetCompressorDictSize 24
 ; The name of the installer
 Name "${APP_NAME}"
 
-OutFile "output\pcsx2-${APP_VERSION}-${OUTFILE_POSTFIX}.exe"
+; Output the installer to the nsis folder (in git repo)
+OutFile "pcsx2-${APP_VERSION}-${OUTFILE_POSTFIX}.exe"
```

Comment:
The only way to create a directory at compile time in MakeNSIS (remember we can't use functions) is to run some code through command line; so it's rather pointless in that matter.

Not sure about the gitignore though, maybe it'll make more sense after I get some rest ;)